src/Sulu/Bundle/WebsiteBundle/EventListener/RedirectExceptionSubscriber.php

```diff
@@ -106,9 +113,16 @@ public function redirectTrailingSlashOrHtml(GetResponseForExceptionEvent $event)
     /**
      * Redirect partial and redirect matches.
      */
-    public function redirectPartialMatch(GetResponseForExceptionEvent $event)
+    public function redirectPartialMatch(ExceptionEvent $event)
     {
-        if (!$event->getException() instanceof NotFoundHttpException) {
+        if (method_exists($event, 'getThrowable')) {
+            $exception = $event->getThrowable();
+        } else {
+            // remove this when symfony 4.3 support is dropped
```

Comment:
Didn't we decide that we drop Symfony 4.3 support?

Reply:
Did create an own PR to have it as own line int he changelog: #5238